### DISCUSSION TITLE
No inventory or blends drop down.

### DISCUSSION THREAD

**Discussion by roastedott** [artisan-roaster-scope/artisan#1746] [Q&A]
Any idea why the roast properties suddenly do not show any of my current inventory that is shown 
<img width="866" alt="Screenshot 2024-11-19 at 2 01 37 PM" src="https://github.com/user-attachments/assets/640f6da5-a4f3-4eb7-bede-070cc56d740e">
in the artisan.plus system.

**Comment by MAKOMO** [comment]
Your Artisan is not logged into the artisan.plus platform. Press the plus icon on the top left of the Artisan main window and put in your credentials when asked.

**Comment by roastedott** [comment]
Marko,
Thank you, I did figure that out finally.  Just took a while and looking back through the install documents.  Just not sure why it was logged out to begin with.

Thank you,

Lori Ott
